livereveal/main.css

```diff
@@ -221,7 +221,7 @@
   font-size: 11px;
   background-color: #f7f7f7;
   color: #333;
-  box-shadow: 0 1px 0 rgba(0, 0, 0, 0.2), 0 0 0 2px #ffffff inset;
+  box-shadow: 0 1px 0 rgba(0, 0, 0, 0.2), 0 0 0 2px #fff inset;
```

Comment:
Also it's strange because in other places of the same file it's #fff and here it's #ffffff. Maybe it was a lessc < 2.5.3 bug? If you want I can revert this.